com.ibm.streamsx.topology/opt/python/packages/streamsx/scripts/streamtool.py

```diff
@@ -390,13 +402,16 @@ def main(args=None):
     """ Mimic streamtool using the REST api for ICP4D.
     """
     streamsx._streams._version._mismatch_check('streamsx.topology.context')
+    val = None
+    rc = 0
     try:
-        sr = run_cmd(args)
-        sr['return_code'] = 0
+        val = run_cmd(args)
+        rc = 0
     except:
-        # print(sys.exc_info())
-        sr = {'return_code':1, 'error': sys.exc_info()}
-    return sr
+        rc = 1
+        sys.exc_info()
+        # sr = {'return_code':1, 'error': sys.exc_info()}
+    return [rc, val]
```

Comment:
The concept was for `run_cmd` to return two values (as a `tuple`) not a `list` of two values.